src/languages/en-US.ts

```diff
@@ -3423,6 +3423,29 @@ export default class extends Language {
 			ESRB: 'ESRB'
 		},
 		COMMAND_ESHOP_PRICE: price => price > 0 ? `$${price} USD` : 'Free',
+		COMMAND_HOROSCOPE_DESCRIPTION: 'Get your latest horoscope',
+		COMMAND_HOROSCOPE_EXTENDED: builder.display('horoscope', {
+			extendedHelp: 'Gets the horoscope for a given sun sign from Kelli Fox\'s The Astrologer.',
+			explainedUsage: [
+				['sunsign', 'The sun sign you want to get the horoscope for'],
+				['today|tomorrow|yesterday', '(Optional, defaults to "today") If you want to get the horoscope of yesterday or tomorrow you can specify that.']
+			],
+			examples: ['pisces', 'virgo tomorrow', 'gemini yesterday', 'aries today']
+		}),
+		COMMAND_HOROSCOPE_INVALID_SUNSIGN: sign => `${sign} is an invalid sun sign, maybe try ${createPick([
```

Comment:
Maybe make an utility that accesses to a random entry and make createPick use it? So we avoid allocating a callback every time this is run (plus `createPick([])()` looks weird)